src/pages/ItemsList/FilterBox/ThemeBtnModal.js

```diff
@@ -0,0 +1,106 @@
+import React, { useState } from 'react';
+import { useNavigate } from 'react-router-dom';
+import styled from 'styled-components';
+import CheckBox from '../CheckButton/CheckBox';
+import THEME_BTN_DATA from './ThemeBtnData';
+
+const ThemeBtnModal = ({ makeModal, setThemeName }) => {
+  const navigate = useNavigate();
+  const [themeData, setThemeData] = useState([]);
+  const [themeId, setThemeId] = useState();
+  const [themeNames, setThemeNames] = useState();
+
+  const checkCheckBox = ({ target }) => {
+    if (target.checked) {
+      setThemeData([target.id]);
+    }
+  };
+
+  const handlerThemeInfo = (themeId, themeName) => {
+    setThemeId(themeId);
+    setThemeNames(themeName);
+  };
+
+  const postThemeId = () => {
+    const themeQuery = { themeId } === 0 ? 'planets' : 'theme=' + themeId;
+    navigate('?' + themeQuery);
+    makeModal();
+    setThemeName(themeNames);
+  };
+
+  return (
+    <Layout>
+      <div className="themeLayout">
+        <div>테마</div>
+        <CloseBtn onClick={() => makeModal()}>X</CloseBtn>
```

Comment:
특별히 인자로 넘길 것이 없다면 아래와 같이 작성해도 좋겠네요!
```suggestion
        <CloseBtn onClick={makeModal}>X</CloseBtn>
```

Reply:
리뷰달아주셔서 감사합니다 😭